FIJI/BatchStitchPooledCellPainting_StitchAndCrop_Headless.py

```diff
@@ -207,8 +245,10 @@
                 rows = str(len(row_widths))
                 columns = str(max(row_widths))
 
-                top_rows = str(int(rows)/2)
-                left_columns = str(int(columns)/2)
+                # xoffset_tiles and yoffset_tiles can be used if you need to adjust the "where to draw the line between quarters" 
+                # by a whole tile. You may want to add more padding if you do this
```

Comment:
Which input changes the padding?

Reply:
padding is set by the final_tile_size * tileperside